examples/azure-functions-deployment/azure_functions_deployment.md

```diff
@@ -1,9 +1,22 @@
 # Azure Functions
 
-Azure Functions is an event driven, serverless computing platform.
-Ballerina functions can be deployed in Azure Functions by annotating a Ballerina function with Azure functions annotation". You can view the code examples below.
-For more information, see the [Azure Deployment Guide](https://ballerina.io/learn/deployment/azure-functions/).
+Azure Functions is an event driven, serverless computing platform. Ballerina functions can be deployed in Azure Functions by annotating a Ballerina function with Azure functions annotation. 
+
+The Trigger, Input and Output bindings parameter annotations were introduced as per the Azure functions programming model. More information can be found on [Azure Deployment Guide](/learn/deployment/azure-functions/).
```

Comment:
```suggestion
The `Trigger`, `Input`, and `Output` bindings parameter annotations were introduced as per the Azure functions programming model. For more information, see [Azure deployment](/learn/deployment/azure-functions/).
```
Check and add the exact name of the Learn page as the hypertext. Need to check a d fix all occurrences of it.